.cloud-build/cleanup/resource_cleanup_manager.py

```diff
@@ -156,3 +157,29 @@ class BatchPredictionJobCleanupManager(VertexAIResourceCleanupManager):
 
 class ExperimentCleanupManager(VertexAIResourceCleanupManager):
     vertex_ai_resource = aiplatform.Experiment
+
+class BucketCleanupManager(VertexAIResourceCleanupManager):
```

Comment:
Buckets are not VertexAI resources so it should subclass `ResourceCleanupManager`

Reply:
fixed